.github/workflows/tests.yml

```diff
@@ -108,6 +108,6 @@ jobs:
     - name: Finished
       run: |
         pip3 install --upgrade coveralls
-        coveralls --finish
+        coveralls --service=github --finish
```

Comment:
@jidicula Any reason why we would not want to put `--service=github` here? I noticed ivadomed only added it to the one above. Ivadomed's PR is linked in the description of this PR.

Reply:
I *think* it should be fine. Honestly coveralls is a bit of voodoo for me - I only remember having a lot of trouble getting this set up with the matrix build and just luckily coming across the right package to use, as the GitHub Actions use-case wasn't well-documented at the time.

Reply:
In any case, checks are passing now so at least we have that confirmation 🎉